include/gamepad/gamepad.hpp

```diff
@@ -90,9 +128,22 @@ class Gamepad {
          * @return std::string A unique listener name
          */
         static std::string unique_name();
-        static Button Gamepad::*button_to_ptr(pros::controller_digital_e_t button);
+        static Button Gamepad::* button_to_ptr(pros::controller_digital_e_t button);
         void updateButton(pros::controller_digital_e_t button_id);
+
+        void updateScreens();
+
+        std::shared_ptr<DefaultScreen> defaultScreen = std::make_shared<DefaultScreen>();
+        std::vector<std::shared_ptr<AbstractScreen>> screens {};
+        ScreenBuffer currentScreen {};
+        ScreenBuffer nextBuffer {};
```

Comment:
using an assignment operator makes this more readable imo

```suggestion
        std::vector<std::shared_ptr<AbstractScreen>> screens = {};
        ScreenBuffer currentScreen = {};
        ScreenBuffer nextBuffer = {};
```